### DISCUSSION TITLE
Check for Updates

### DISCUSSION THREAD

**Discussion by arvanus** [mariuz/flamerobin#218] [Ideas]
What do you think about a functionality to check manually (or automatically) for updates? 
If "yes" should we use github API to get info about new releases/snapshots?

**Comment by luronumen** [comment]
That would be perfect! :D

**Comment by mariuz** [comment]
implemented by https://github.com/mariuz/flamerobin/commit/7445ce7dcd11b5c013d6ef6c79b0d568fb879633
